packages/lib-user/src/components/Contributors/helpers/generateExport.js

```diff
@@ -0,0 +1,61 @@
+import { arrayOf, func, number, shape, string } from 'prop-types'
+
+import { getExportData } from './getExportData'
+
+const DEFAULT_HANDLER = () => true
+
+export function generateExport({
+  group,
+  handleFileName = DEFAULT_HANDLER,
+  handleDataExportUrl = DEFAULT_HANDLER,
+  projects,
+  stats,
+  users
+}) {
+  const data = getExportData({ projects, stats, users })
+
+  let str = ''
+
+  data.forEach((row) => {
+    str += row.map((col) => JSON.stringify(col)).join(',').concat('\n')
+  })
+
+  // The following regexp sanitizes the group name by removing all non-alphanumeric characters (i.e. emojis, spaces, punctuation, etc.)
+  let sanitizedGroupName = group.display_name.replace(/[^a-zA-Z0-9]/g, '')
+
+  let newFilename = `${sanitizedGroupName}.data_export.${Date.now()}.csv`
```

Comment:
`Date.now()` works, and for example I successfully exported data with the file name `TestGroup5.data_export.1718129145512.csv`. Just to double check though, `Date.now()` is not really human-readable. Was the intent to insert a date into the filename to simply make it unique? Or to use a human-readable date like YYYY_MM_DD? For example, lib-content now uses `dayjs` for easy conversion in CommunityContainer if you're looking for a lightweight third-party package.

Reply:
Intent was just to make filename unique, but I think the date would be helpful, especially if easy to do with `dayjs`. I'll confirm including date is preferable with Sean/Laura and update accordingly 👍 .